contracts/TokenStore.sol

```diff
@@ -51,9 +51,10 @@ contract Ownable {
     _;
   }
 
-  function transferOwnership(address _newOwner) onlyOwner {
```

Comment:
Please revert that part, the old one is better.

Reply:
@tokenstoredev 

There's tow 'old' versions: 

1) https://github.com/tokenstore/contract/commit/e61c9dd91db9ce8785f109cf0206b7e2565f4284#diff-7953b479a014e7309c8088e71766cf16L54

2) https://github.com/tokenstore/contract/commit/e61c9dd91db9ce8785f109cf0206b7e2565f4284#diff-8fff83b4119641a960ebf126d3729954L15

Which one we use in both cases? Or, should we leave 2 different versions in 2 different files as it was before?